src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -211,7 +213,7 @@
 
 (defn message-not-sent-text
   [chat-id message-id]
-  [rn/touchable-highlight
+  [react/touchable-highlight
```

Comment:
this for example

Reply:
Oh it just need to be rebased

Reply:
could you rebase it please ? so we could review

Reply:
looks like you made chages in old ui ns, here still no changes

Reply:
The latest commit changes is only in messages.cljs under ui2

Reply:
Urgh… I am not sure what happened, will check again